tests/micro/zephyr/test_zephyr_aot.py

```diff
@@ -233,6 +233,7 @@ def test_qemu_make_fail(temp_dir, platform, west_cmd, tvm_debug):
     xx = relay.multiply(x, x)
     z = relay.add(xx, relay.const(np.ones(shape=shape, dtype=dtype)))
     func = relay.Function([x], z)
+    func = tvm.IRModule.from_expr(func)
```

Comment:
Given this isn't the function anymore, maybe we should go with `ir_mod` or `mod` ?